packages/ethereum-contracts/contracts/superfluid/SuperTokenFactory.sol

```diff
@@ -175,7 +185,7 @@ abstract contract SuperTokenFactoryBase is
         }
 
         if (upgradability == Upgradability.NON_UPGRADABLE) {
-            superToken = ISuperToken(this.createSuperTokenLogic(_host));
+            revert SUPER_TOKEN_FACTORY_NON_UPGRADEABLE_IS_DEPRECATED();
```

Comment:
NOTE: non upgradeable super token deployment via SuperTokenFactory is deprecated now.